org.eclipse.xtext.builder/META-INF/MANIFEST.MF

```diff
@@ -43,7 +43,7 @@ Export-Package: org.eclipse.xtext.builder,
  org.eclipse.xtext.builder.trace;x-friends:="org.eclipse.xtext.xbase.ui,org.eclipse.xtend.ide",
  org.eclipse.xtext.builder.trace.impl;x-internal:=true,
  org.eclipse.xtext.builder.trace.util;x-internal:=true
-Require-Bundle: org.eclipse.xtext,
+Require-Bundle: org.eclipse.xtext;bundle-version="2.17.0",
```

Comment:
We usually do not maintain plugin versions.

Reply:
@AndersDala please remove this

Reply:
Will do!